creator/events/models.py

```diff
@@ -93,6 +94,14 @@ class Meta:
         max_length=1000,
         help_text="Description of the event",
     )
+    organization = models.ForeignKey(
+        Organization,
+        null=False,
```

Comment:
Yeah this is a bit annoying since we have to make sure we can always link back to an organization when we create an event. The current logic is that the organization is pretty vital to determining whether someone has permission to view an event or not. If there were events that did not have organizations attached, it would only be visible to super user types. I would tend to think that these sorts of events would be more appropriate to log through logging or Sentry but maybe this requirement will change if we have some really high-level admin automation and tooling that needs to happen in the future :shrug: .